src/ScaleIdentification.jl

```diff
@@ -0,0 +1,107 @@
+using MIDI
+
+
+"""
+    MIDI_to_notes
+
+Returns the note names of an array of MIDI pitch value, inside of single octave.
+
+Example :
+julia> Midi_to_notes([0,13,26])
+["C","C","C"]
+
+"""
+function MIDI_to_notes(MIDInotes::Array{})
+    name = ["C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B"]
+    MIDIrange = []
+    for i in range(-1,12)
+        for note in name
+            push!(MIDIrange,note)
+        end
+    end
+    notes = []
+    for n in MIDInotes
+        push!(notes,MIDIrange[n+1])
+    end
+    return notes
+end
+
+
+"""
+    most_frenquent_notes(notes)
+
+Takes an Array representing musical notes
+and returns the 7 most frequent values from this array.
+
+"""
+function most_frenquent_notes(notes::Array{})
+    occurency = sort(Dict(value => key for (key, value) in countmap(notes)), rev = true)
+    most_frequent = Any[n for n in values(occurency)]
```

Comment:
It is not correct to have `Any` as the container type. It is bad for performance and bad for reading code. Why is `Any`? Don't you know what kind of data it contains? If you do, then use this data type e.g. `String` whatever it is.

Reply:
Actually scratch that. Why do you even do that? Don't you see that `most_frequent` is just identical with `values(occurency)`? Why do you do all this transformation? Just write:
```
 most_frequent = values(occurency)
```
???